src/apps/containers.clj

```diff
@@ -18,7 +19,8 @@
         [korma.core :exclude [update]]
         [korma.db :only [transaction]])
   (:require [clojure.tools.logging :as log]
-            [korma.core :as sql]))
+            [korma.core :as sql]
+            [clojure.string :as string]))
```

Comment:
It's not consistent everywhere. In apps there are places where I started mimicking the import order used in our UI code, so `clojure.*` comes first, 3rd party libs next, then `apps.*` at the bottom, but I think that's more confusing in clojure than just alphabetical order 😖